src/utils/utils.ts

```diff
@@ -71,4 +71,8 @@ export const utils = {
         }
         return chunkedItems;
     },
+    delay: async (ms: number): Promise<{}> => {
+        // tslint:disable-next-line
```

Comment:
Don't use blanket tslint disables. Always disable specific linter rules. What are you disabling here?

Reply:
`tslint:disable-next-line:no-inferred-empty-object-type`